tests/unit/test_config.py

```diff
@@ -103,7 +103,7 @@ def test_environment_configuration_not_found(computing_environment):
 
 
 @pytest.mark.parametrize(
-    "key, input",
+    "key, value",
```

Comment:
All changes in this file are trivial - just rename the args so we're not overloading `input`